molgenis-api-data/src/main/java/org/molgenis/api/data/v1/SortV1.java

```diff
@@ -17,10 +17,8 @@
 
 import java.io.Serializable;
```

Comment:
FYI The class docs state that ```Copy of org.springframework.data.domain.Sort for backwards compatibility```. I checked whether the [current implementation](https://github.com/spring-projects/spring-data-commons/blob/master/src/main/java/org/springframework/data/domain/Sort.java) is still similar to ours, because in that case I would favor suppressions. But since the current implementation is quite different from ours I prefer your solution.